kie-wb-common-react/boxed-expression-editor/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "boxed-expression-editor",
+  "main": "server.js",
+  "private": true,
+  "scripts": {
+    "prebuild": "rimraf dist",
+    "build": "webpack --config webpack.prod.js",
+    "start": "webpack serve --hot --color --progress --config webpack.dev.js",
+    "start:prod": "node server.js",
+    "test": "jest",
+    "lint": "eslint --ext .tsx,.js ./src/",
+    "type-check": "tsc --noEmit",
+    "quality-checks": "npm run type-check && npm run lint && npm run test"
+  },
+  "devDependencies": {
+    "@testing-library/jest-dom": "^5.11.5",
+    "@types/enzyme": "^3.10.7",
+    "@types/enzyme-adapter-react-16": "^1.0.6",
+    "@types/jest": "^26.0.14",
+    "@types/react-dom": "^16.9.9",
+    "@types/react-router-dom": "^5.1.6",
+    "@types/victory": "^33.1.4",
+    "@typescript-eslint/eslint-plugin": "^4.4.1",
+    "@typescript-eslint/parser": "^4.4.1",
+    "copy-webpack-plugin": "^6.2.1",
+    "css-loader": "^5.0.0",
+    "dotenv-webpack": "^4.0.0",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.5",
+    "enzyme-to-json": "^3.6.1",
+    "eslint": "^7.11.0",
+    "eslint-plugin-react": "^7.21.4",
+    "eslint-plugin-react-hooks": "^4.1.2",
+    "express": "^4.17.1",
```

Comment:
Personally, I believe it's a bit odd to have `express` here. Alternatively, we could try to use [React-scripts](https://github.com/karreiro/kogito-guided-tour/blob/master/dev-webapp/package.json) (?). I don't think it's blocking for the merge tho ;-)

Reply:
Just for information. Let me explain a bit the purpose of `express` presence. I am using it (only as `devDependency`) just to have the chance to test the production artifact (so, after the `build` phase has been performed). But it is not a mandatory step. I will remove it in the next PR.